src/pressure_sensor/include/pressure_sensor/pressure_sensor.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <rclcpp/rclcpp.hpp>
+#include <std_msgs/msg/float64.hpp>
+#include <std_msgs/msg/string.hpp>
+
+#define DEFAULT_DUMP_PARAM 1.0
+
+namespace stingray_core::pressure_sensor {
+
+class PressureSensorNode {
```

Comment:
Нодой по факту он уже и не является
```suggestion
class PressureSensor {
```